modules/overview/pages/introduction-to-eclipse-che.adoc

```diff
@@ -120,18 +120,18 @@ For situations where the default IDE does not cover the use cases of the users o
 
 {prod} is extensible in different ways:
 
-* *Plug-ins* to add capabilities to the IDE. Che-Theia plug-ins rely on APIs compatible with Visual Studio Code. Plug-ins are isolated and provide their own dependencies packaged in containers.
+* *Plug-ins* to add capabilities to the IDE. Che-Theia plugins rely on APIs compatible with Visual Studio Code. Plug-ins are isolated and provide their own dependencies packaged in containers.
```

Comment:
Remove this line?

Reply:
That's the task @deerskindoll is currently working on, I will stay on scope.

Reply:
Ah, apologies then.